docs/01-usage/configuration.md

```diff
@@ -85,55 +112,101 @@ The git repository URL.
 
 Any valid git url format is supported (See [Git protocols](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols)).
 
-### tagFormat
+Example (`.releaserc` file content):
+```json
+{
+  "repositoryUrl": "https://github.com/username/project.git"
+}
+```
+
+### `tagFormat`
 
 Type: `String`<br>
 Default: `v${version}`<br>
 CLI arguments: `-t`, `--tag-format`
 
 The [Git tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) format used by **semantic-release** to identify releases. The tag name is generated with [Lodash template](https://lodash.com/docs#template) and will be compiled with the `version` variable.
 
-**Note**: The `tagFormat` must contain the `version` variable exactly once and compile to a [valid Git reference](https://git-scm.com/docs/git-check-ref-format#_description).
+> **Note**: The `tagFormat` must contain the `version` variable exactly once and compile to a [valid Git reference](https://git-scm.com/docs/git-check-ref-format#_description).
 
-### plugins
+Example (`.releaserc` file content):
+```json
+{
+  "tagFormat": "version-${version}"
+}
+```
+
+### `plugins`
 
 Type: `Array`<br>
 Default: `['@semantic-release/commit-analyzer', '@semantic-release/release-notes-generator', '@semantic-release/npm', '@semantic-release/github']`<br>
 CLI arguments: `-p`, `--plugins`
 
-Define the list of plugins to use. Plugins will run in series, in the order defined, for each [steps](../../README.md#release-steps) if they implement it.
-
-Plugins configuration can defined by wrapping the name and an options object in an array.
+Define the list of plugins to use. Plugins will run in series, **in the order defined**, for **each [steps](../../README.md#release-steps)** if they implement it.
 
-See [Plugins configuration](plugins.md#plugins) for more details.
+Plugin specific configuration can defined by wrapping the name and an options object in an array. See [Plugins configuration options](plugins.md#plugins-configuration-options) for more details.
 
-### dryRun
+Example (`.releaserc` file content):
+```json
+{
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    "@semantic-release-docker",
+    ["@semantic-release/exec", {
+      "verifyConditionsCmd": "./verify.sh"    # plugin configuration
```

Comment:
Unfortunately, comments are not allowed in JSON. Adding them will make it harder to copy & paste the examples.

We could add a description below explaining the different bits of it?

Reply:
Done in next push. Explanation already in the text.